src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -192,8 +194,8 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
 
     return (
       this.state.shippingOption === "SHIP" &&
-      processWithArtaShipping &&
-      shippingCountry === "US"
+      ((processWithArtaShipping && shippingCountry === "US") ||
+        (artsyShippingInternational && shippingCountry !== "US"))
```

Comment:
Just for my own learning 🧠 :

Am I correct in saying will be relying on validations on the CMS side of things to handle checks of where the artwork is located? Assuming the protection to only allow US exports live there?

Reply:
Yes!
This logic will have to be changed when new countries are included into Artsy Shipping, we will have to check artwork location against user address...
Though nothing blocks us to do it now, I think I'll give it a try

Reply:
No presh! we can handle that when we get to it